canary/sanity.go

```diff
@@ -42,8 +42,8 @@ var sanityChildWFList = []string{
 	wfTypeCancellation,
 	wfTypeRetry,
 	wfTypeReset,
-	wfTypeHistoryArchival,
-	wfTypeVisibilityArchival,
+	//wfTypeHistoryArchival,
```

Comment:
why are you commenting these out?  These can be excluded through config.

Reply:
They are not registered and generate errors. Should I register them instead?

Reply:
FWIW I see errors from `wfTypeVisibilityArchival` even after I've added it to the excludes list.

Reply:
I now understand that these will fail if the cluster does not have archiving enabled 😄

Reply:
Yes register them instead of commenting it out.  There is already config based mechanism to exclude them.